### ISSUE TITLE
Broken highlighting- no option to reset the environment

### ISSUE THREAD

**Issue by edolis** [continuedev/continue#3109]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: Win 10
- Continue version: v0.8.59
- IDE version: Vscode 1.95.3
- Model: Codestral
- config.json:
```

### Description

After some activity, often happens that the highlighting gets messed up and I have found no way to clear that up.
Closing and opening the file does not help.
Sometimes the accept/reject option does not appear, and as you try to delete suggested code manually, the highligthing stays fixed and everything turns into an utter nightmare. Pressing the ctrlShiftenter/esc does not help.
![image](https://github.com/user-attachments/assets/e75381c9-0f2d-4a49-a971-9b79bfbf131d)

**Comment by sestinj** [comment]
@edolis We are working on fixing the buttons not showing up. There should be updates in a pre-release soon. Otherwise, you can use cmd+shift+backspace/enter to reject/accept all changes

**Comment by edolis** [comment]
thanks, unfortunately as I wrote that does not fix it. Now I am thinking of renaming the file and deleting it, but having it in source control is just making things so painful.
also thought for asking a review of the whole code, but then I get an endless and arbitrary list of suggestions to remove my comments and so on. Which is another issue.
Together to the issue that the AI (codestral) sometimes abbreviates blocks of code and that is being pushed as a proposed deletion. Every time over and over again.
But these are other issues you can manage with local inserts. But which invalidate the use of asking full code review to clear that formatting issue.

**Comment by edolis** [comment]
Ah, OK, found a workaround. Need to restart VScode.

**Comment by edolis** [comment]
yes, that's a common problem. I can use it only on source control where I can compare version and roll back afterwards selectively. 
So use only the insert at position option (and do the clean ups manually afterwards).
The arrow to apply the full code is workable only if, from the beginning, you have the whole code generated by the AI (so in very limited cases of small code chunks). It won't understand your commenting and using variable to extract parameters and so on

**Comment by sestinj** [comment]
This is good feedback. I think it might be helpful in the future for us to add the ability to open a virtual diff view instead of directly writing to the editor, or at least give you the option to switch to this after the initial diff is shown

**Comment by olm** [comment]
I'm facing this exact problem in PHPSTORM. Restarting also helps
